### PR TITLE
Update legislation.tex

### PR DESCRIPTION
**Pull request by mep-clearhaus**
New and up-to-date version of the cake legislation.

OBS, I need someone to be create a PDF version of this whenever merged.

New stuff
- Added both Quickpay and 3DSecure.io, and their employees, as potential COR. 
- Removed item relating to Dinero
- Added "Size"/"Amount" in relation *Cake*
- Adding a rule relating to the POP2 Padel Court
- Removing superfluous paragraphs to ensure we are being served a little less cake than the old legislation was permitting

### CODE REVIEW

File: legislation.tex [hunk 1]
```diff
@@ -29,8 +29,8 @@
 \chapter{Legislative basics}
 
 \begin{enumerate}[§ 1]
-  \item This legislation applies to all employees of Clearhaus {\small A/S} and
-    any subsidiary companies in Denmark, as well as abroad.
+  \item This legislation applies to all employees of Clearhaus, Quickpay, and 3DSecure.io.
```
Comment (ct-clearhaus) [inline legislation.tex]:
```suggestion
  \item This legislation applies to all employees of Clearhaus, Quickpay, and 3dsecure.io.
```

Capitalisation cf. https://datacvr.virk.dk/enhed/virksomhed/41019328.

File: legislation.tex [hunk 2]
```diff
@@ -68,16 +70,13 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Using incorrect recipient in sufficiently important email.
   \item Having passed an exam, either final exam or getting the top grade.
   \item Extreme weather\footnote{DMI Category 2 Warning in the Aarhus area.} (\emph{cake responsible organization} is Clearhaus A/S).
-  \item The household gets a tax refund larger than 10,000 DKK\@.
   \item Winning a programming competition.
   \item Returning from parental leave.
-  \item Full-time employment in Clearhaus A/S or any other company.
+  \item Going from student position, internship or similar non-full-time-employment to Full-time employment in Clearhaus, Quickpay, or 3DSecure.io.
```
Comment (ct-clearhaus) [inline legislation.tex]:
```suggestion
  \item Going from student position, internship or similar non-full-time-employment to full-time employment in Clearhaus, Quickpay, or 3dsecure.io.
```

Consequence.

**Review by ct-clearhaus — DISMISSED** [review]
Nice! 🎾

Should we add also 🏓 while we're at it? 🤔

**Comment by mep-clearhaus** [comment]
> Should we add also 🏓 while we're at it? 🤔

Yes!

**Comment by cbobach** [comment]
We were considering if we should add that `Clearhaus gives cake if network is down` 🤷🏻

**Comment by brorbw** [comment]
Something like `If wifi down > 15 min CH will provide cake`. This will also make up for missed cake due to removal of sections

**Comment by Lassejoe** [comment]
> wifi down > 15 min

Or `if Wi-Fi down multiple times same day`
Or `if Wi-Fi down due to new settings`

**Comment by mep-clearhaus** [comment]
I've tried to add the wifi disturbance input :D Please check it out

File: legislation.tex [hunk 3]
```diff
@@ -83,5 +83,6 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately)
+  \item If wifi down > 15 min, or/and wifi down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```
Comment (cbobach) [inline legislation.tex]:
```suggestion
  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```

**Review by cbobach — COMMENTED** [review]
Suggestion 🤷🏻

File: legislation.tex [hunk 4]
```diff
@@ -68,20 +70,19 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
...
-  \item Adding the Dinero mailing list to a newsletter.\\
-    {\tiny (What are you doing?? You work at Clearhaus!?)}
   \item Shameful defeat (0--7) in foosball.
+  \item Shameful defeat (0--6) in padel.
+  \item Shameful defeat (0--11) in table tennis.
   \item Code deployment causing service outage.
   \item Forgetting that you are on call.
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately)
```
Comment (kbadk) [inline legislation.tex]:
```suggestion
  \item Occupying a meeting room booked by another person (without leaving immediately).
```

File: legislation.tex [hunk 5]
```diff
@@ -68,20 +70,19 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
...
+  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```
Comment (kbadk) [inline legislation.tex]:
```suggestion
  \item If network down > 15 min, and/or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```

File: legislation.tex [hunk 6]
```diff
@@ -83,6 +83,6 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately).
-  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
+  \item If network down > 15 min, and/or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```
Comment (brorbw) [inline legislation.tex]: `or` should suffice. Its ambiguous if both conditions should apply /s
```suggestion
  \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```

**Review by brorbw — CHANGES_REQUESTED** [review]

**Review by brorbw — APPROVED** [review]

**Pull request merged**